uchar.c

```diff
@@ -520,14 +584,16 @@ int u_to_ascii(char *dst, const char *src, int len)
 	return i;
 }
 
-void u_to_utf8(char *dst, const char *src)
+const char *u_to_utf8(char *dst, const char *src)
```

Comment:
would you mind explaining this signature change? It is not obvious to me why it is needed.

Reply:
Looks like it is used to allow for chaining, which makes the code look a lot nicer.

Reply:
Yeah, it's just for chaining. There's no need for it beyond aesthetics.

Reply:
I don't like this change, as it is purely aesthetic and chaining is not a widely used idiom in the cmus code base.